Dockerfile

```diff
@@ -3,6 +3,14 @@ FROM --platform=$BUILDPLATFORM node:20-slim AS builder
 # Build application and add additional files
 WORKDIR /ghostfolio
 
+RUN apt-get update && apt-get install -y --no-install-suggests \
```

Comment:
I've moved the package installation step because it's less likely to change so the cache will be invalidated less frequently